masonite/view.py

```diff
@@ -51,3 +75,120 @@ def extend(self):
     def share(self, dictionary):
         self.dictionary.update(dictionary)
         return self
+
+    def cache_for(self, time=None, type=None):
+        """
+        Set time and type for cache
+        """
+
+        self.cache_time = time
+        self.cache_type = type
+
+        if self.__is_expired_cache():
+            self.__create_cache_template(self.template)
+        return self
+
+    # --------
+    # Privates
+    # --------
+
+    @property
```

Comment:
This should not be a property. It works when it is not a property and just a normal method. Is there a specific reason this is a property?

this will cause a headache because it is the only property and it should be a method. This will make maintenance harder because it's harder to debug a parenthesis issue. Especially because its valid when it is missing but has a different behavior than normal.

Reply:
Ok!